doc/final-design.md

```diff
@@ -151,10 +157,19 @@ Get module binary using account address.
 
 # Design architecture
 
+The main parts are:
+- PalletMove _(this repo)_.
+- The `no-std` MoveVM fork adapted for the Substrate framework.
+  - It can be found inside the _[substrate-move][substrate-move]/language/_ directory.
+- The backend layer which is an interface between the MoveVM and the pallet.
+  - It is located inside the _[substrate-move][substrate-move]/_ directory.
+- [smove][smove] tool, which is necessary for the compilation of Move source code.
```

Comment:
It's really necessary for **the compilation**?

Reply:
Agreement from Slack - let's keep it like this to avoid confusion and extra info